tests/webhook_test.go

```diff
@@ -183,6 +184,21 @@ var _ = Describe("Validation webhook", func() {
 				err := apiClient.Create(ctx, newSspV1Beta3, client.DryRunAll)
 				Expect(err).To(MatchError(ContainSubstring("missing name in DataImportCronTemplate")))
 			})
+
+			It("should fail if .spec.cluster is nil and multi-arch is enabled", func() {
+				newSspV1Beta3.Spec.EnableMultipleArchitectures = ptr.To(true)
+				newSspV1Beta3.Spec.Cluster = nil
+				err := apiClient.Create(ctx, newSspV1Beta3, client.DryRunAll)
+				Expect(err).To(MatchError(ContainSubstring(".spec.cluster needs to be non-nil, if multi-architecture is enabled")))
+			})
+
+			It("should fail if control plane architectures are empty", func() {
+				newSspV1Beta3.Spec.Cluster = &sspv1beta3.Cluster{
+					ControlPlaneArchitectures: nil,
+				}
+				err := apiClient.Create(ctx, newSspV1Beta3, client.DryRunAll)
+				Expect(err).To(MatchError(ContainSubstring(".spec.cluster.controlPlaneArchitectures cannot be empty")))
```

Comment:
Not sure about this. We have clusters without control plane nodes, e.g. hypershift clusters. Anyway, it is possible anf we can't reject ssp changes for that.

Reply:
done